WordPress/Classes/Models/AbstractPost.h

```diff
@@ -13,6 +13,7 @@ typedef NS_ENUM(NSUInteger, AbstractPostRemoteStatus) {
     AbstractPostRemoteStatusLocal,       // Only local version
     AbstractPostRemoteStatusSync,       // Post uploaded
     AbstractPostRemoteStatusPushingMedia, // Push Media
+    AbstractPostRemoteStatusSaved,       // Post saved
```

Comment:
If this is specifically refering to autosaved posts, can we rename it to something like `AbstractPostRemoteStatusAutoSaved`?

It's a minor difference, but "saving" can a bit vague and be confused with the regular uploading of the post, whereas "autosaving" has a more concrete meaning (at least in my head).

Reply:
Also, if possible, update the comment to be "Post auto-saved" as well.